docs/pages/link.js

```diff
@@ -430,8 +432,20 @@ Accessible content is critical if we consider that assistive technology also pre
           title="Keyboard navigation"
           description="Give Link keyboard focus with the tab key (or shift + tab when tabbing backwards). Activate Link with the enter/return key."
         />
+
         <MainSection.Subsection
-          title="Accessible Tab Link"
+          title="External links"
+          description={`An external link, also called an outbound link, is a link from Pinterest to a different website.
+
+When rendering an external Link, add text for screen readers to announce that Link will go to a different destination. Users should be informed that they will be moving out of a domain and which domain they are moving to. This is particularly relevant for those with cognitive impairments or people relying on assistive technology.
+
+When possible, limit one external Link per paragraph, as adding more than two icons in the same block of text can clutter the design and create readability issues.
+
+For external Links that aren't in a paragraph or text context, consider [Button's link role](https://gestalt.netlify.app/button#Role) or [IconButton's link role](https://gestalt.netlify.app/iconbutton#Role).`}
```

Comment:
rewording for clarity: "…consider Button or IconButton with `role="link"`."

Reply:
DONE